src/tractusx_sdk/dataspace/tools/http_tools.py

```diff
@@ -39,28 +39,28 @@
 class HttpTools:
 
     # do get request without session
-    def do_get(url,verify=True,headers=None,timeout=None,params=None,allow_redirects=False):
+    def do_get_without_session(url,verify=True,headers=None,timeout=None,params=None,allow_redirects=False):
```

Comment:
```suggestion
    @staticmethod
    def do_get_without_session(url,verify=True,headers=None,timeout=None,params=None,allow_redirects=False):
```
The reason this CodeQL warning appears is that this method is static but is written like a normal method.